### ISSUE TITLE
Group search without ou

### ISSUE THREAD

**Issue by Nix3c3r** [django-auth-ldap/django-auth-ldap#272]
I'm trying to perform a group search withot an ou in the dn string:

```
AUTH_LDAP_GROUP_SEARCH" = LDAPSearch(
        "dc=catalysis,dc=de",
        ldap.SCOPE_SUBTREE,
        "(objectClass=group)"
),
AUTH_LDAP_GROUP_TYPE = ActiveDirectoryGroupType(),
```

This failes with:
`"search_s('dc=catalysis,dc=de', 2, '(&(objectClass=group)(member=cn=harald viemann,ou=dlinke,ou=dlinke,ou=bereiche,ou=institut,dc=catalysis,dc=de))') raised OPERATIONS_ERROR({'msgtype': 100, 'msgid': 5, 'result': 1, 'desc': 'Operations error', 'ctrls': [], 'info': '000004DC: LdapErr: DSID-0C090A71, comment: In order to perform this operation a successful bind must be completed on the connection., data 0, v3839'})"`

If I add in any ou the search does its job, but I want to search all available ous I tried to do a short debug, but can't see why it does not work. Considering the server return I would guess that the user isn't bound correctly during the group search. On the other hand the search wit an ou added schould fail as well. The user is bound via the user search, which uses an alternative fixed user. The login woks fine. Do you have an idea?

**Comment by francoisfreitag** [comment]
Could it be the following question from the [Python LDAP FAQ](https://www.python-ldap.org/en/python-ldap-3.4.0/faq.html):

> My script bound to MS Active Directory but a a search operation results in the exception [ldap.OPERATIONS_ERROR](https://www.python-ldap.org/en/python-ldap-3.4.0/reference/ldap.html#ldap.OPERATIONS_ERROR) with the diagnostic message text “In order to perform this operation a successful bind must be completed on the connection.” Alternatively, a Samba 4 AD returns the diagnostic message “Operation unavailable without authentication”. What’s happening here?

**Comment by dalito** [comment]
From a look a the FAQs I think it could be another item. 

@francoisfreitag You can close this issue if you want. @Nix3c3r was working with me when the ticket was created but moved on to new challenges.

**Comment by Nix3c3r** [comment]
I agree with @dalito. In case you want to resolve it you can keep it open, otherwise I'me fine with it being closed.

**Comment by francoisfreitag** [comment]
Thanks for the input. Just wanted to check whether the FAQ entry could have been the cause for the issue, in case the search without `ou` returned referrals, causing the operations error.
If not, then let’s leave the issue open, to raise awareness.
